core/lib/dal/migrations/20231026192041_applied-snapshot-status.down.sql

```diff
@@ -0,0 +1,3 @@
+-- Add down migration script here
+
+DROP TABLE IF EXISTS applied_snapshot_status;
```

Comment:
nit: please set up your editor so that it edds newline in the end of files on save

Reply:
done